job-service-db/src/main/resources/procedures/getJobs.sql

```diff
@@ -183,6 +184,46 @@ BEGIN
          ELSE quote_ident(in_sort_field)
        END ||
         ' ' || CASE WHEN in_sort_ascending THEN 'ASC' ELSE 'DESC' END;
-    RETURN QUERY EXECUTE sql;
+
+    -- Create temporary table as a base to update the job progress
+    EXECUTE 'CREATE TEMPORARY TABLE get_job_temp ON COMMIT DROP AS ' || sql;
```

Comment:
I've just realized there could be a problem here with the status and percentage complete ordering planned for [SCMOD-10977](https://portal.digitalsafe.net/browse/SCMOD-10977).  I think we can address it in another PR rather than pursuing it now.